client/src/main.ts

```diff
@@ -7,7 +7,7 @@ const game = {
 	type: Phaser.AUTO,
 	width: 1280, // TODO: Figure out how big this game should be
 	height: 720,
-	parent: 'moogiwara',
+	parent: 'MOOgiwara',
```

Comment:
I think for any internal usage of phaser naming, we will be using all lowercase naming

Reply:
I was under the assumption that this is referring to the HTML tag parent
is this a module name?

Reply:
i don't actually know what this `parent` is